sympy/stats/drv_types.py

```diff
@@ -74,6 +75,14 @@ def DiscreteRV(symbol, density, set=S.Integers):
     set : set
         Represents the region where the pdf is valid, by default is real line.
 
+    Other Parameters
+    ================
```

Comment:
We shouldn't have a section called "Other parameters".

The expected sections are listed here:
https://docs.sympy.org/latest/documentation-style-guide.html#docstring-sections

Reply:
I have added it under the "Parameters" section in that case. 

I found an error in the guide: they state that one needs an extra section for `Returns` but `Returns` is not part of the list of the valid sections. Probably not a big deal though.